checkstyle.xml

```diff
@@ -43,7 +43,9 @@
 
     <!-- Checks whether files end with a new line.                        -->
     <!-- See http://checkstyle.sf.net/config_misc.html#NewlineAtEndOfFile -->
-    <module name="NewlineAtEndOfFile"/>
+    <module name="NewlineAtEndOfFile">
```

Comment:
Unrelated changed, but required for checkstyle on windows. Not sure why this did not trigger previously.